content/features/featuresDeepDive/babylonViewer.md

```diff
@@ -42,15 +42,13 @@ The Babylon.js viewer automatically provides a [default viewing experience](#def
 
 ## Example
 
-<p data-height="400" data-theme-id="light" data-slug-hash="QxzBPd" data-default-tab="html,result" data-user="BabylonJS" data-embed-version="2" data-pen-title="Babylon.js Viewer - Display a 3D model" class="codepen">See the Pen <a href="https://codepen.io/BabylonJS/pen/QxzBPd/">Babylon.js Viewer - Display a 3D model</a> by Babylon.js (<a href="https://codepen.io/BabylonJS">@BabylonJS</a>) on <a href="https://codepen.io">CodePen</a>.</p>
+<CodePen pen="QxzBPd" user="BabylonJS" tab="html,result" title="Babylon.js Viewer - Display a 3D model" />

```

Comment:
The component doesn't have a user prop, right? I think hardcoding to BabylonJS to make sure everyone is using the same account is good.

Reply:
Removed. I experimented with it, but haven't removed it. This will technically force us to only have codepen's from our user, but it is actually safer so I am fine with it.